build/azure-pipelines/darwin/product-build-darwin.yml

```diff
@@ -93,64 +93,64 @@ steps:
       yarn gulp vscode-reh-web-darwin-min-ci
   displayName: Build
 
-- script: |
-    set -e
-    ./scripts/test.sh --build --tfs "Unit Tests"
-  displayName: Run unit tests (Electron)
-  condition: and(succeeded(), eq(variables['VSCODE_STEP_ON_IT'], 'false'))
-
-- script: |
-    set -e
-    yarn test-browser --build --browser chromium --browser webkit --browser firefox
-  displayName: Run unit tests (Browser)
-  condition: and(succeeded(), eq(variables['VSCODE_STEP_ON_IT'], 'false'))
-
-- script: |
+#- script: |
```

Comment:
I know these comments are only temporary, but you can always change the `VSCODE_STEP_ON_IT` to anything but `false` when submitting a build, to skip all of these steps.